pkg/gui/views.go

```diff
@@ -121,6 +121,13 @@ func (gui *Gui) createAllViews() error {
 			return err
 		}
 	}
+	frameRunes := []rune{'─', '│', '┌', '┐', '└', '┘'}
+	switch gui.c.UserConfig.Gui.Border {
+	case "double":
+		frameRunes = []rune{'═', '║', '╔', '╗', '╚', '╝'}
```

Comment:
need to update: https://github.com/jesseduffield/gocui/blob/eeb9b3fb24940a469698e835b2076acb622c8e73/tcell_driver.go#L24

Reply:
@yk-kd could you raise a PR in the gocui repo adding these newly added runes to the `runeReplacements` variable (as @Ryooooooga linked above)?

Reply:
@jesseduffield Is this not enough?
https://github.com/jesseduffield/gocui/pull/27